configyaml/config/__init__.py

```diff
@@ -1,5 +1,5 @@
-from .base import ConfigBase
-from .base_list import ConfigBaseList
-from .base_dict import ConfigBaseDict
-from .base_wildcard_dict import ConfigBaseWildcardDict
-from .base_choice import ConfigBaseChoice
+from .base import AbstractNode
+from .list import NodeList
+from .dict import NodeDict
+from .wildcard_dict import WildcardDict
+from .choice import NodeChoice
```

Comment:
Seems fine to me. Why `WildcardDict` instead of `NodeWildcardDict`? You can also rename the `config` dir if that makes sense too.